### ISSUE TITLE
Debugger restart button not working anymore

### ISSUE THREAD

**Issue by eamodio** [Microsoft/vscode#57714]
Issue Type: <b>Bug</b>

After starting a debug session (debugging an extension) and try to restart the session by clicking the restart button in the UI it results in this error:
![image](https://user-images.githubusercontent.com/641685/44933959-2167ec00-ad39-11e8-82cd-c3e29ed66153.png)

Also the tooltip on that button is "Reconnect" now, rather than "Restart" so my guess that is part of the problem. I have also seen the stop button turn into the disconnect button as well.

This all just started in one of the very recent insiders builds.

VS Code version: Code - Insiders 1.27.0-insider (1097b5e9ed81d5f9241648774b35048be168c203, 2018-08-31T10:38:54.277Z)
OS version: Windows_NT x64 10.0.17746

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz (8 x 4008)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|31.93GB (13.37GB free)|
|Process Argv|C:\Users\Eric\AppData\Local\Programs\Microsoft VS Code Insiders\Code - Insiders.exe|
|Screen Reader|no|
|VM|0%|

</details>

**Comment by haschu** [comment]
Got the same error and found this to be related: https://github.com/Microsoft/vscode/issues/57018
That workaround solved it for me: https://github.com/Microsoft/vscode/issues/57018#issuecomment-415183598

**Comment by eamodio** [comment]
@haschu I don't think this is related. This issue just started when I first reported, and now exists in the newly released 1.27.1 stable release as well.

**Comment by eamodio** [comment]
I think this is also caused by Live Share as well.

**Comment by weinand** [comment]
@rodrigovaras fyi

**Comment by rodrigovaras** [comment]
Yes, live share debugging support was relying on the presence of this property during launch/attach. We are working on a fix to use the public API to avoid future breaking changes. The patch will be available soon
